stack/logging.rst

```diff
@@ -22,6 +26,13 @@ Logger Names
 
 Logger names should generally start with the fully qualified name of the module/file containing the logger, without the leading ``lsst.``.
 Some example logger names are ``afw.image.MaskedImage`` and ``meas.algorithms.starSelector``.
+A common Python recommendation is to create the logger name from the module hierarchy automatically:
+
+.. code-block:: python
+
+   log = logging.getLogger(__name__.partition(".")[2])
+
+**(Aside to be removed: moving all the logging support code to utils should allow us to have a helper function for this -- an argument could be made that it's better to include the ``lsst.`` prefix to allow simpler global log configuration. It should also allow a proper Rubin getLogger that can return a thing that supports verbose/trace).**
```

Comment:
The use of Python logging that can conflict with logging by other packages basically mandates that we use `lsst.`, doesn't it?  Want to RFC that?

Reply:
I can RFC that and see what happens.

Reply:
RFC-801